core/survey/_survey/surveyAnalysis.js

```diff
@@ -55,12 +55,22 @@ export const getAnalysisNodeDefs =
         if (!hasAnalysisSibilings) return false
       }
 
-      if(hideAreaBasedEstimate && NodeDef.isAreaBasedEstimatedOf(nodeDef)){
+      if (hideAreaBasedEstimate && NodeDef.isAreaBasedEstimatedOf(nodeDef)) {
         return false
       }
 
-      if(!showInactiveResultVariables && !NodeDef.getActive(nodeDef)){
-          return false 
+      if (!showInactiveResultVariables && !NodeDef.getActive(nodeDef)) {
+        return false
+      }
+
+      if (
+        Boolean(NodeDef.getAreaBasedEstimatedOf(nodeDef)) &&
```

Comment:
Boolean(NodeDef.getAreaBasedEstimatedOf(nodeDef)) can be replaced with isAreaBasedEstimatedOf.. isn't it?